src/OpenTelemetry.Exporter.Console/README.md

```diff
@@ -17,12 +17,15 @@ dotnet add package OpenTelemetry.Exporter.Console
 ```
 
 See the
-[`TestConsoleExporter.cs`](../../examples/Console/TestConsoleExporter.cs) for an
-example of how to use the exporter for exporting traces.
-
 See the [Program](../../docs/logs/getting-started/Program.cs) for
 an example of how to use the exporter for exporting logs.
 
+See the [Program](../../docs/metrics/getting-started/Program.cs) for
```

Comment:
Consider changing these to bullet points:

See the individual "getting started" examples depending on the signal being used:
* Logs - [/docs/logs/getting-started/Program.cs](../../docs/logs/getting-started/Program.cs)
* Metrics - [/docs/metrics/getting-started/Program.cs](../../docs/metrics/getting-started/Program.cs)
* Traces - [/docs/traces/getting-started/Program.cs](../../docs/traces/getting-started/Program.cs)